integration-tests/src/test/java/oracle/kubernetes/operator/ItUsabilityOperatorHelmChart.java

```diff
@@ -659,42 +581,14 @@ public void testCreateWithEmptyTargetDomainInstall() throws Exception {
   }
 
   /**
-   * Helm installs the operator with default target domains namespaces.
-   *
-   * @throws Exception exception
-   */
-  //@Test -commenting out, it fails for runs in parallel due sharing same targetDomainNS.
-  // uncomment if want to run in single run
-  public void testCreateWithDefaultTargetDomainInstall() throws Exception {
-    Assumptions.assumeTrue(QUICKTEST);
-    String testMethodName = new Object() {
-    }.getClass().getEnclosingMethod().getName();
-    logTestBegin(testMethodName);
-    Operator operator = null;
-    try {
-      Map<String, Object> operatorMap = createOperatorMap(getNewSuffixCount(), true, "usab");
-      ArrayList<String> targetDomainsNS = new ArrayList<String>();
-      targetDomainsNS.add("default");
-      operatorMap.replace("domainNamespaces", targetDomainsNS);
-      operator = new Operator(operatorMap, true, true, false, RestCertType.SELF_SIGNED);
-      operator.callHelmInstall();
-      operator.verifyOperatorReady();
-
-    } finally {
-      if (operator != null) {
-        operator.destroy();
-      }
-    }
-    LoggerHelper.getLocal().log(Level.INFO, "SUCCESS - " + testMethodName);
-  }
-
-  /**
-   * Create operator and verify its deployed successfully. Create domain1 and verify domain is
-   * started. Call helm upgrade to add domainnew to manage, verify both domains are managed by
-   * operator Call helm upgrade to remove first domain from operator target domains, verify it can't
-   * not be managed by operator anymore.
+   * Create operator and verify its deployed successfully. 
+   * Create domain1 and verify domain is started. 
+   * Upgrade the operator to new domain doamin2 to be managed by the Operator
```

Comment:
domain spelled wrong

Reply:
what does this sentence even mean? are you updating the target domainNamespaces?

Reply:
operator should not have capital

Reply:
Modified the description